### DISCUSSION TITLE
Prior austenite grain boundary for pearlite or bainite/ferrite?

### DISCUSSION THREAD

**Discussion by pxyg** [mtex-toolbox/mtex#1430] [Ask Anything] [answered]
Dear all,

I have a short question about prior austenite grain boundary reconstruction. Now mtex can work with martensite parent grain reconstruction. Does that also works for other microstructures, such as pearlitic or ferritic/bainitic steel?

Thanks!

**Comment by nyyssont** [comment, marked as answer]
Short answer is that you might expect the reconstruction to work for displacive transformations, such as the martensite or bainite transformation. Diffusional transformations, such as pearlite or ferrite formation, will not result in grains that can be reconstructed into a parent structure.
